docs/src/main/sphinx/object-storage.md

```diff
@@ -34,21 +35,17 @@ in a catalog. Each catalog can only use one file system.
 * - Property
   - Description
 * - `fs.hadoop.enabled`
-  - Activate the [legacy libraries and implementation based on the Hadoop](file-system-legacy)
-    ecosystem. Defaults to `true`.
+  - Activate the [legacy libraries and implementation based on the
+    Hadoop](file-system-legacy) ecosystem. Defaults to `false`.
```

Comment:
Suggest moving this down to the bottom of the list since we want to discourage its use